spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/kafka-streams.adoc

```diff
@@ -173,6 +173,11 @@ Default: `earliest`.
 Note: Using `resetOffsets` on the consumer does not have any effect on Kafka Streams binder.
 Unlike the message channel based binder, Kafka Streams binder does not seek to beginning or end on demand.
 
+concurrency::
+  Number of stream task threads. Needs at least these many number of partitions in the consuming topic.
++
+Default: `1`.
+
```

Comment:
Why is this here? `concurrency` is a common binding property, not a kafka (streams)-spedific property: [Common Binding Properties](https://docs.spring.io/spring-cloud-stream/docs/Elmhurst.SR1/reference/htmlsingle/#_common_binding_properties).

Reply:
It certainly doesn't belong in this section, which talks about `spring.cloud.stream.kafka.streams.bindings.<binding name>.consumer.'literal`.

Perhaps add a note somewhere that the common concurrency property maps to the `num.stream.threads` kafka property.

BTW, there's something wrong with the `spring.cloud.stream.kafka.streams.bindings.<binding name>.consumer.'literal.` above.

I am not sure what "literal" means here, but asciidoc doesn't render properly when a back-ticked phrase isn't followed by a space.

Reply:
Sure; fine with me; we don't talk about how it's mapped for other binders.